auth-agents-common/src/main/java/org/apache/atlas/policytransformer/CachePolicyTransformerImpl.java

```diff
@@ -122,6 +122,7 @@ public ServicePolicies getPolicies(String serviceName, String pluginId, Long las
 
             AtlasEntityHeader service = getServiceEntity(serviceName);
             servicePolicies.setPolicyVersion(-1L);
+            servicePolicies.setPolicyUpdateTime(new Date());
```

Comment:
new Date() will take the server timezone. Is that OK ? If yes, I'm good with this.

Reply:
This date will be in sync with the entity audits date since both are generated in Atlas server